tests/CMakeLists.txt

```
@@ -296,4 +304,5 @@ set_property(TARGET tests_Namespace_life_cycle PROPERTY CXX_STANDARD 20)
 add_test(NAME tests_Namespace_life_cycle COMMAND tests_Namespace_life_cycle)
 
 # copy files for testing to the binary folder
-# file(COPY foldername DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
\ No newline at end of file
+# file(COPY foldername DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+file(COPY ./parser/tests_RDFFileParser_simple.ttl DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

```suggestion
file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/parser/tests_RDFFileParser_simple.ttl" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
```

In CMake, `${ ... }` should be used within a string. 

The second thing is probably more a matter of taste: please use an absolute path here. Double check that the suggestion above really works (i.e. delete the cmake folder and run cmake again).